README.md

```diff
@@ -1,4 +1,6 @@
-# NUbots Codebase [![Build status](https://badge.buildkite.com/85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.svg?branch=master)](https://buildkite.com/nubots/nubots)
+# NUbots Codebase
+
+[![Overall Build Status](https://img.shields.io/buildkite/85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40/master?label=overall)](https://buildkite.com/nubots/nubots) [![Build & Test Status](https://img.shields.io/badge/dynamic/json?style=flat&label=build%20%26%20test&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DBuild%2520generic%2520and%2520Test)](https://buildkite.com/nubots/nubots) [![C++ Format Status](https://img.shields.io/badge/dynamic/json?style=flat&label=c%2B%2B%20format&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DValidate%2520C%252B%252B%2520and%2520Protobuf%2520formatting)](https://buildkite.com/nubots/nubots) [![Cmake Format Status](https://img.shields.io/badge/dynamic/json?style=flat&label=cmake%20format&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DValidate%2520CMake%2520formatting)](https://buildkite.com/nubots/nubots) [![Python Format Status](https://img.shields.io/badge/dynamic/json?style=flat&label=python%20format&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DValidate%2520Python%2520formatting)](https://buildkite.com/nubots/nubots) [![Nusight Build Status](https://img.shields.io/badge/dynamic/json?style=flat&label=nusight%20build&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DBuild%2520NUsight2)](https://buildkite.com/nubots/nubots) [![Nusight Test Status](https://img.shields.io/badge/dynamic/json?style=flat&label=nusight%20test&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DTest%2520NUsight2)](https://buildkite.com/nubots/nubots) [![Nusight Format Status](https://img.shields.io/badge/dynamic/json?style=flat&label=nusight%20format&query=%24.status&url=https%3A%2F%2Fbadge.buildkite.com%2F85cb206a2615c85981c4e0089b0abb0c6bcd775b3d946ede40.json%3Fbranch%3Dmaster%26step%3DCheck%2520NUsight%2520code%2520with%2520eslint)](https://buildkite.com/nubots/nubots)
```

Comment:
Is it worth wrapping these in a table?

Reply:
something like this?

# Build Status
| Type | status |
| ---- | ---- |
| NUbots | _generic_ _nuc1_ _nuc2_ |
| NUbook | _build_ _storybook_ |
| Formatting and Style | _f1_ _f2_ _f3_ _f4_ |

I don't think I like that, but I am guessing as to what you're actually intending

Reply:
# Build Status
| Project | Building | Test | Formatting |
| ---- | ---- | ---- | ---- |
| NUbots | _generic_ _nuc1_ _nuc2_ |
| NUbook | _build_ _storybook_ |

Maybe something like this? Im not sold on this idea either, just thinking about adding aome structure so its not just a mass of badges

Reply:
Honestly, it might even be too extra to have more than the overall ones and maybe like `Build generic and Test` now that I have seen how many badges we can have :joy:

Reply:
Maybe. Im open to other ideas.

Reply:
Not sold on the table. I am hink it's OK as is, but perhaps we could show the few main ones first and drop the rest in a collapsible `<details>` section that you can click to see.

Reply:
That works for me

Reply:
The badges break in a details tag :(

Reply:
You need newlines before and after the markdown when you put it in an HTML tag, otherwise it's interpreted as HTML and not compiled.